build/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 0.7.0-dev
+
+A number of changes to the apis, primarily to support reading/writing as bytes,
+as this is going to inevitably be a required feature. This will hopefully be the
+last breaking change before the `1.0` release, but it is a fairly large one.
```

Comment:
I am going to leave it in for now, as I think we should wait to release until we have all breaking changes done. I set up a milestone for this (0.7.0) to track other changes, so we can continue to add to the changelog for this version.